googlemock/README.md

```diff
@@ -28,6 +28,15 @@ gMock:
 -   does not use exceptions, and
 -   is easy to learn and use.
 
+
+Details and examples can be found here:
+
+
+*   [gMock for Dummies](https://github.com/google/googletest/blob/master/googlemock/docs/for_dummies.md)
```

Comment:
Could you please use relative url. For example "googlemock/docs/for_dummies.md"

Reply:
Thank you. Fixed.